src/main/java/com/denizenscript/denizencore/objects/core/ElementTag.java

```diff
@@ -1128,17 +1128,16 @@ else if (elem.equals("false")) {
         // @returns ElementTag
         // @group element checking
         // @description
-        // Returns the character at a specified index.
+        // Returns the character at a specified index, supporting negative indexes to count from the end.
         // Returns null if the index is outside the range of the element.
         // -->

```

Comment:
Might be nice to add an example here, just to make sure the way it works is easily understandable